mmv1/third_party/terraform/tpgresource/utils_test.go

```diff
@@ -1079,6 +1079,32 @@ func TestReplaceVars(t *testing.T) {
 			},
 			Expected: "projects/default-project/zones/default-zone/instances",
 		},
+		"location with provider level region": {
+			Template: "projects/{{project}}/locations/{{location}}/repositories",
+			Config: &transport_tpg.Config{
+				Project: "default-project",
+				Region:  "default-region",
+			},
+			Expected: "projects/default-project/locations/default-region/repositories",
+		},
```

Comment:
q: can these unit tests be run directly from within magic modules (and are they run on any of the CI runs in this project)? I think maybe the provider `build-and-unit-test` steps only test the unit tests within the provider itself?

The only way I could seem to get them to run, whether with `make` or `go test` was first building the provider and then running `go test -v ./google/tpgresource` from the base of the repo... got import issues building them directly within `magic-modules`

Reply:
They cannot be run directly from magic modules, unfortunately. 

`build-and-unit-test` is run in the generated provider.

Reply:
Thanks. And I think from the logs, it doesn’t run these tests, maybe, right?

Reply:
The tests should be run in tpgresource folder https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/11824442763/job/32945960561

Reply:
Got it, yeah, I looked there, just don't see anything if I search the logs for `tpgresource`, but could just be missing it.

I ran it locally anyway, was just curious if they really ran in CI somewhere too.